container/root/tests/php-fpm/beta.goss.yaml

```diff
@@ -44,6 +44,10 @@ package:
     installed: true
   php7.1-zip:
     installed: true
+  librdkafka-dev:
+    installed: true
+    versions: 
+      - "0.9.1-1ppa1~xenial"
```

Comment:
if this version gets updated, its going to break any further builds. Can you do a substring to just look for `0.9.1` instead?